packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/gux-dropdown-multi.tsx

```diff
@@ -69,6 +69,14 @@ export class GuxDropdownMulti {
   @Prop()
   hasError: boolean = false;
 
+  /**
+   * allows dropdown popup to be wider than input
+   * defaults to fitting content if width is not specified for listbox
+   * default min-width is set to width of input
+   */
+  @Prop()
+  exceedtargetwidth: boolean = false;
```

Comment:
changes lgtm 👍  minor but I think this should be `exceedTargetWidth`.

Is this to be backported also ?

Reply:
Do we need a comment on the case of the prop in the code? Just since it's kind of an anomaly as far as I know

Reply:
Maybe.. I'm not sure why this one was weird yet hasError seems to work? unless the difference is it being set programmatically with JS or something vs in the dom

Reply:
It works ok for me if I change it to camelCase and then update the attribute in the example to exceed-target-width. Can you give me any more info on recreating the error?